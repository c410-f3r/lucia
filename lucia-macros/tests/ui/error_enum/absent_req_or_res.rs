#[lucia_macros::pkg]
mod a {
}

#[lucia_macros::pkg]
mod b {
  #[pkg::req_data]
  struct Req(
    i32
  );
}

#[lucia_macros::pkg]
mod c {
  #[pkg::res_data]
  struct Res;
}

fn main() {
}
