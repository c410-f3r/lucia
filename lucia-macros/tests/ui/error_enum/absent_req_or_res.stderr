error: The `#[pkg]` module must have an inner `#[pkg::req_data]` element and an inner `#[pkg::res_data]` element.
 --> tests/ui/error_enum/absent_req_or_res.rs:2:5
  |
2 | mod a {
  |     ^

error: The `#[pkg]` module must have an inner `#[pkg::req_data]` element and an inner `#[pkg::res_data]` element.
 --> tests/ui/error_enum/absent_req_or_res.rs:6:5
  |
6 | mod b {
  |     ^

error: The `#[pkg]` module must have an inner `#[pkg::req_data]` element and an inner `#[pkg::res_data]` element.
  --> tests/ui/error_enum/absent_req_or_res.rs:14:5
   |
14 | mod c {
   |     ^
